src/routes/faq.js

```diff
@@ -0,0 +1,68 @@
+const express = require('express');
+const router = express.Router();
+const {Faq} = require('../models');
+const {QUESTION_MAX_LENGTH, ANSWER_MAX_LENGTH} = require('../utils/variables');
+
+router.get('/', (req, res) => {
+	Faq
+		.findAll()
+		.then((faqs) => { res.status(200).json(faqs) });
```

Comment:
if you pass only one argument you can amend brackets ```.then(faqs => //...)```

Also if you make only oneline function you can amend curly brackets
so it will be ```.then(faqs => res.status(200).json(faqs));```

Reply:
where is catch?